docs/status.md

```diff
@@ -107,7 +107,7 @@ one of the following tracking labels.
 | real                     | no            | yes*         | yes*           | yes             | no          |
 | real_dynamic_slice       | no            | revisit      | no             | yes             | no          |
 | recv                     | no            | revisit      | no             | no              | no          |
-| reduce                   | no            | yes*         | yes*           | revisit         | no          |
+| reduce                   | yes           | yes          | yes            | revisit         | no          |
```

Comment:
I was wrong to initially recommend to flip verification to "yes". While we're figuring out #369, I think it should be "revisit".